crt_portal/cts_forms/templates/partials/footer.html

```diff
@@ -82,6 +85,9 @@
           <a href="https://www.justice.gov/doj/budget-and-performance">
             {% trans "Budget and performance" %}
           </a>
+          <a href="https://www.justice.gov/legalpolicies">
+            {% trans "Office of Legal policies and Disclaimers" %}
```

Comment:
Noting we don't have a translation for this but it appears on https://www.justice.gov/espanol
Políticas Legales y Descargos de Responsabilidades

Though we have "office of" which isn't on the DOJ site

Reply:
Yes, this came up in review sprint -- will need to remove "Office of"

Reply:
cleared this with Dan at sprint planning :+1:

Reply:
https://github.com/usdoj-crt/crt-portal/pull/498/commits/f25266cea9c18910e6b65b8f3ef90b6e28099d99